packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -113,12 +113,25 @@ class MemoryController {
 
     for (Map instance in instances) {
       final String objectRef = instance['id'];
-      result.add(InstanceSummary(classRef, objectRef));
+      result.add(InstanceSummary(classRef, className, objectRef));
     }
 
     return result;
   }
 
+  Future<Instance> getObject(String objectRef) async {
```

Comment:
why isn't this using serviceManager.service.getObject?

Reply:
Thought there was a problem and wrote it directly and forgot to undo.
It's use getObject.